lib/datastore/request.js

```diff
@@ -382,75 +384,68 @@ DatastoreRequest.prototype.save = function(entities, callback) {
 
   var insertIndexes = [];
 
-  var req = {
-    mutation: entities.reduce(function(acc, entityObject, index) {
-      entityObject = extend(true, {}, entityObject);
+  var mutation = {
+    insert: [],
+    update: [],
+    upsert: [],
+    insert_auto_id: []
+  };
 
-      var ent = {};
-      var method = entityObject.method;
-      delete entityObject.method;
```

Comment:
Should we keep this? I realize the overall size difference will be small, but this should make our payload a little smaller, yes?